src/libAtomVM/mailbox.c

```diff
@@ -22,6 +22,9 @@
 #include "memory.h"
 #include "scheduler.h"
 #include "synclist.h"
+
+//#define ENABLE_TRACE
+
```

Comment:
cleanup: let's remove this, so we can make the changeset smaller

Reply:
Done.